frontend/src/Editor/Editor.jsx

```diff
@@ -171,6 +180,32 @@ class Editor extends React.Component {
     });
   };
 
+  fetchOrgEnvironmentVariables = () => {
+    orgEnvironmentVariableService.getVariables().then((data) => {
+      const client_variables = {};
+      const server_variables = {};
+      data.variables.map((variable) => {
+        if (variable.variable_type === 'server') {
+          server_variables[variable.variable_name] = 'SecureValue';
```

Comment:
If we plan to keep server values as a secret, should do it from the server-side itself. As these values could otherwise be obtained from the network responses

Reply:
@akshaysasidrn ^

Reply:
Understood 👍🏾